### DISCUSSION TITLE
Call js after wire:init

### DISCUSSION THREAD

**Discussion by toriqsetiawan** [livewire/livewire#4521] [1. Help] [answered]
Can someone help me how to call js function after ```wire:init``` in Livewire?
there are any event or something to handle this?
thanks

**Comment by joshhanley** [comment, marked as answer]
@toriqsetiawan in whatever function you call from `wire:init` I would just dispatch a browser event and then listen for that in your JS.

`wire:init="doSomething"`

```php
public function doSomething()
{
    $this->dispatchBrowserEvent('something-happened');
}
```

Hope this helps!
